### ISSUE TITLE
Impossible to post the same /me message twice in a row with new double-space workaround

### ISSUE THREAD

**Issue by zneix** [Chatterino/chatterino2#3141]
**Describe your issue**
After #3081 there's a bug that looks like this:

1. Type `/me test`
1. Press `Ctrl+Enter` once to send it (it goes through).
1. Press `Ctrl+Enter` once to send it again (it fails).

This occurs because on the second message Chatterino tries to send `/me<Space><Space>test`, which (I think) gets interpreted as a message with content `test`, but that's exactly what you've sent in your previous message, thus it can't go through.

**Screenshots**
N/A (actually too lazy to take them)

**OS and Chatterino Version**
Arch GNU/Linux 5.13.8, 54f6e4f

**Comment by Mm2PL** [comment]
![2021-08-07_16-30](https://user-images.githubusercontent.com/25011746/128603592-12f7b181-da0d-47eb-94aa-81b52fc97cbe.png)

**Comment by zneix** [comment]
Thanks for providing the screenshot I was too lazy to take.

**Comment by zneix** [comment]
Also, if you don't mind, I assigned myself because I feel like fixing this one. Hope it's ok.

**Comment by Mm2PL** [comment]
Go for it then. Double spaces were my idea so I felt obliged to fix this. FeelsDankMan

**Comment by Felanbird** [comment]
I've been informed that this has not technically been fixed in cases where only the chatterino invis character is used, or any other invis character.
![image](https://user-images.githubusercontent.com/41973452/130569904-73d7dd6f-da9e-4da9-9557-4d5e16001f3b.png)
![image](https://user-images.githubusercontent.com/41973452/130569954-e099ba81-db79-4739-9680-038282ee42ee.png)

```
[2021-08-24 06:44:40] #pajlada felanbird: 󠀀
[2021-08-24 06:44:41] #pajlada felanbird: 󠀀 󠀀
[2021-08-24 06:44:42] #pajlada felanbird: 󠀀
[2021-08-24 06:44:43] #pajlada felanbird: 󠀀 󠀀
[2021-08-24 06:44:44] #pajlada felanbird: 󠀀
[2021-08-24 06:44:59] #pajlada felanbird: i can do it like that
```

The specifics of this seems to be, when attempting to send me you have to type `/me` with a space, and then paste the chatterino invis character, but when you copy it from chat usually spaces are included, even if you are extremely careful with the copying.

When copy pasting the character into any unicode checker (I used https://www.babelstone.co.uk/Unicode/whatisit.html ), and stripping all spaces, re-copying the invis character and then typing `/me 󠀀` space and unicode character included, I'm once again able to spam.

![image](https://user-images.githubusercontent.com/41973452/130572402-01855c31-a60c-48aa-94c1-d8aa8bef1e6e.png)

**Comment by Mm2PL** [comment]
Example: `/me`, space, space, `.`. This gives the same results as you described but is simpler and doesn't rely on weird copying.

**Comment by Mm2PL** [comment]
Fixed again in #3312
